### PR TITLE
[Security][Http] Remove BC layers

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by chalasr** [comment]
Green

File: src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php [hunk 1]
```diff
@@ -45,17 +41,11 @@ public function __construct($listener)
         }
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function __invoke(RequestEvent $event)
     {
         $startTime = microtime(true);
         if (\is_callable($this->listener)) {
```
Comment (OskarStark) [inline src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php]: Is this if check still needed?

Comment (chalasr) [reply src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php]: fixed thanks

File: src/Symfony/Component/Security/Http/Firewall.php [hunk 2]
```diff
@@ -38,17 +36,15 @@ class Firewall implements EventSubscriberInterface
...
-        // the type-hint will be updated to the "EventDispatcherInterface" from symfony/contracts in 5.0
-
         $this->map = $map;
         $this->dispatcher = $dispatcher;
         $this->exceptionListeners = new \SplObjectStorage();
     }
 
     /**
-     * @internal since Symfony 4.3
+     * @internal
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Http/Firewall.php]: what about removing `@internal` on methods now? We don't have to keep it, do we?

**Review by fabpot — CHANGES_REQUESTED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @chalasr.

**Pull request merged**